connectivity/FEATURE_BLE/include/ble/common/Bounded.h

```diff
@@ -47,9 +47,9 @@ struct Bounded {
     Bounded(Rep v) : _value(v)
     {
         if (v < Min) {
-            _value = v;
+            _value = Min;
```

Comment:
this is unexpected in a "refactor"

Reply:
Ahah, little bug I caught, it should have had its own commit.